config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get ":slug/next", to: "redirections#next"
```

Comment:
It decouples the site that it came from the site that it's going to, which could also be a security disadvantage, i.e. anyone could use someones slug and start the webring. Although I dunno what the advantage of that would be.

Also, this allows people to have multiple subdomains that hook into the webring, i.e. `example.com` and `blog.example.com` could both use the `example` slug, instead of having multiple sites listed for each redirection or multiple sites from the same domain in the ring.

Also, we don't have to parse URLs either, even though that seems pretty solid, there could be weird edge cases.

Seems more flexible, if not as slick with the same link for everyone.